app/lib/dartdoc/backend.dart

```diff
@@ -320,3 +320,58 @@ class DartdocBackend {
     await deletePool.close();
   }
 }
+
+class ArchiveTracker {
+  final _pvMap = <String, _TrackedEntry>{};
+  final _sampleLines = <String>[];
+
+  ArchiveTracker() {
+    _logger.info('Tracking: init.');
```

Comment:
Suggestion: This message could be more specific and mention ArchiveTracker? It's also not in the same format as the  lines printed out by _logSamples. It doesn't say which tracker it is.